### PR TITLE
Fixes and tweaks

### PR DESCRIPTION
**Pull request by ElArGee**
- Fixed view var calls
- Cleaned up tick delay defines
- Added hotkeys for dropping items from specific hands
- Added hotkey for swapping intent
- Added verb for resetting keybinds to default

You will need to either reset your keybinds, or manually set the new ones. We may need to implement a client prefs versioning system at some point for dynamically resetting this stuff when these sorts of changes are made.

New defaults:
Drop left item: Q
Drop right item: E
Swap intent: R

### CODE REVIEW

File: code/mobs/mob_inventory.dm [hunk 1]
```diff
@@ -13,6 +13,13 @@
 	UpdateIcon()
 	return TRUE
 
+/mob/proc/DropSlot(var/invslot)
```
Comment (MistakeNot4892) [inline code/mobs/mob_inventory.dm:17]: `GetEquipped (slot)` covers a chunk of this.

Comment (ElArGee) [reply code/mobs/mob_inventory.dm:17]: Oh, nice, I'm a dummy and completely missed that.

File: code/mobs/mob_inventory.dm [hunk 2]
```diff
@@ -13,6 +14,11 @@
...
+	var/obj/item/O = GetEquipped(var/invslot)
```
Comment (MistakeNot4892) [inline code/mobs/mob_inventory.dm]: Declaring a null var in the proc call? :D

**Comment by ElArGee** [comment]
I'm so bad.

**Pull request merged**